### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by SimonGideon**
In this PR, I:
1. Uses media queries in your CSS file to make your desktop version look like the template you selected (Desktop: 768px or wider).
2. Used Grid and Flexbox to layout components on the page.
3. Flexible elements on the page.
4. Fixed the font sizing using view width (vw)

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -1,255 +1,310 @@
...
-            <span><img src="/images/project_1.svg" alt="project_1"></span>
+          <span><img src="/images/project_1.svg" alt="project_1" /></span>
+          <div>
             <span><h2>Tonic</h2></span>
             <div class="compn-lan">
-                <p>CANOPY</p>
-                <ul>
-                    <li>Back End Dev</li>
-                    <li>2015</li>
-                </ul>
```
Comment (aimalamiri) [inline Index.html:63]: Good job! This looks very nice, but you can follow the Figma template to improve the design by removing the extra space between the elements.

image

File: Index.html [hunk 2]
```diff
@@ -1,255 +1,310 @@
...
-        <span><img src="/images/project_2.png" alt="project_2"></span>
-        <span><h2>Multi-Post Stories</h2></span>
-        <div class="compn-lan">
-            <p>CANOPY</p>
-            <ul>
-                <li>Back End Dev</li>
+      </div>
+      <!-- project-2 -->
+      <div class="projects">
+        <div class="card">
```
Comment (aimalamiri) [inline Index.html:103]: As you see in the Figma template, the image for the even cards is on the right, and the image for the odd cards is on the left. 

**Tip: You can use the `nth-child()` function in here. Look at [this](https://www.w3schools.com/cssref/sel_nth-child.asp)**

image

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @SimonGideon, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights

- The desktop version is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Index.html (hunk 2 above)
Comment (SimonGideon) [reply Index.html:103]: @aimalamiri Hello, I have implemented the nth-child and now I can access the odd and even children.
```
  /* reverse the nth child of project */
  .work div:nth-child(even) {
    display: none;
  }
```
Under Class work,  I have 3 projects similar to project-1 given below. My question is how can I access the image div inside and swap their order?
```
<section class="work">
      <!-- project-1 -->
      <div class="project-1 projects">
        <div class="card">
          <div>
            <img src="/images/project_1.svg" alt="project_1"/>
          </div>
          <div>
            <span><h2>Tonic</h2></span>
            <div class="compn-lan">
              <p>CANOPY</p>
              <ul>
                <li>Back End Dev</li>
                <li>2015</li>
              </ul>
            </div>
            <div class="card-body">
              <p>
                A daily selection of privately personalized reads; no accounts
                or sign-ups required.
              </p>
            </div>
            <div class="technology">
              <ul>
                <li>html</li>
                <li>css</li>
                <li>javaScript</li>
              </ul>
            </div>
            <button class="button">See Project</button>
          </div>
        </div>
      </div>
</section>
```

Comment (SimonGideon) [reply Index.html:103]: ## 🎉 :clinking_glasses: After a long struggle made it out of the rubbit hole. :smiley: 
giphy

File: css/styles.css [hunk 3]
```diff
@@ -346,3 +352,223 @@ button:hover {
...
+
+  .social-icons li a img {
+    width: 140%;
+  }
+
+  .social-icons {
+    width: 12rem;
+  }
+
+  /* work-section */
```
Comment (Alfredbis29) [inline css/styles.css:420]: - [ ] Good work on making some changes so far, but you still need to enhance your design for your project to look fantastic, but you're almost there. Please minimize the gap between your work sections to make your project looks like one for Figma design. :wink: 

This is how your work section look like :point_down: 
spacebetweensections

Figma Design :point_down: 
figmaworksection

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
Hi @SimonGideon ,
 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
- Good readMe :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- No linter errors :heavy_check_mark: 

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
-N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
Hi @SimonGideon ,
 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
- Good readMe :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- No linter errors :heavy_check_mark: 

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
-N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SimonGideon** [comment]
@Alfredbis29 Applied the suggested changes to the Section requested by the previous reviewer.

**Review by mgmediaweb — APPROVED** [review]
Hi @SimonGideon , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved5

## Highlights :100
- Good job following the advice of the previous reviewer ✔️ 

## Optional suggestions

[OPTIONAL] Kindly, I suggest you use "A" tags for the main menu (navbar)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**